pkg/resources/provisioning.cattle.io/v1/cluster/validator_test.go

```diff
@@ -131,3 +136,60 @@ func Test_isValidName(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateMachinePoolName(t *testing.T) {
```

Comment:
Feel free to start the test with `t.Parallel()`.

Reply:
You can run the test AND subtests in parallel in this case. The first line of the test can also be `t.Parallel()`.

Reply:
Ahhh fun! Updated.